src/cmds/scripts/pbs_postinstall.in

```diff
@@ -882,28 +913,28 @@ if [ ! -d "$newpbs_exec" ]; then
 	exit 1
 fi
 
-if [ -z "$6" -o "$6" != "sameconf" ]; then
+if [ "$sameconf" != "true" ]; then
```

Comment:
The original code used "$6", but here we use "$sameconf" which is set in the new code based on more than just "$6"...is this intentional?